draft-ietf-gnap-core-protocol.md

```diff
@@ -3276,11 +3324,11 @@ Authorization: Bearer OS9M2PMHKUR64TB8N6BW7OZB8CDFONP219RP1LT0
 The form parameter and query parameter methods of {{RFC6750}} MUST NOT
 be used.
 
-If the "key" value is the boolean `true`, the access token MUST be sent
-to the RS using the same key and proofing mechanism that the client instance used
+If the `key` value is the boolean `true`, the access token MUST be sent
```

Comment:
maybe this should be review when we talk about key rotation, because it might not be the same as the initial

Reply:
This is a good point, I'll add in the rotation language here to be consistent with other parts of the spec.